components/AdminFunc/AllClasses.tsx

```diff
@@ -35,17 +52,26 @@ export default function AllClasses() {
               <TableField>{sportClass.name}</TableField>
               <TableField>{sportClass.teacher}</TableField>
               <TableField>
-                <Link
-                  href={{
-                    pathname: 'admin',
-                    query: {
-                      id: sportClass.id,
-                    },
-                  }}
-                >
-                  Edit
-                </Link>
-                <DeleteClass id={sportClass.id}>Delete</DeleteClass>
+                <ButtonWrapper>
+                  {/* <Link
```

Comment:
Remove the commented code

Reply:
Im still working on that bug from yesterday, will redirect to a new page. Keeping this for when that happens.